cupaloy.go

```diff
@@ -66,10 +62,12 @@ func (c *config) SnapshotMulti(snapshotID string, i ...interface{}) error {
 
 func (c *config) SnapshotT(t *testing.T, i ...interface{}) {
 	t.Helper()
-	snapshotName := strings.Replace(t.Name(), "/", "-", -1)
-	err := c.snapshot(snapshotName, i...)
-	if err != nil {
-		t.Error(err)
+	if !t.Failed() {
```

Comment:
Can this be refactored to keep normal code at minimal indentation? See https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
So just
```go
if t.Failed() {
    return
}
...
...
```